src/pages/SignUp/internal/SignUpConfirm/internal/SignUpConfirm.component.tsx

```diff
@@ -20,23 +20,22 @@ const SignUpConfirm = ({
       descriptionTop={H1_DESCRIPTION_TOP}
     >
       <Stack>
-        <Divider />
         <Stack divider={<Divider />} gap="12px">
           <div className={styles.exampleContainer}>
-            <p>メールアドレス</p>
-            <p>{email}</p>
+            <p className={styles.signup_confirm__label}>メールアドレス</p>
+            <p className={styles.signup_confirm__value}>{email}</p>
           </div>
           <div className={styles.exampleContainer}>
-            <p>年齢層</p>
-            <p>{age_group}</p>
+            <p className={styles.signup_confirm__label}>年齢層</p>
+            <p className={styles.signup_confirm__value}>{age_group}</p>
           </div>
           <div className={styles.exampleContainer}>
-            <p>性別</p>
-            <p>{gender}</p>
+            <p className={styles.signup_confirm__label}>性別</p>
+            <p className={styles.signup_confirm__value}>{gender}</p>
           </div>
           <div className={styles.exampleContainer}>
-            <p>都道府県</p>
-            <p>{prefecture}</p>
+            <p className={styles.signup_confirm__label}>都道府県</p>
```

Comment:
こういう感じ

```suggestion
            <Spacer size="12px" />
            <p className={styles.signup_confirm__label}>都道府県</p>
```

参考: https://github.com/ih14c-13-14/artist-frontend/blob/main/src/pages/SignIn/internal/SignIn.component.tsx